### DISCUSSION TITLE
2021/01/how-to-force-git-to-prompt-you-for-a-password/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#674] [Specific Post Comments]
# 2021/01/how-to-force-git-to-prompt-you-for-a-password/

A fun git challenge! I had to make a request against a remote repository in git. The only issue is that only a shared GitHub account had access, not my own account (due to a vendor limitation). So when I occasionally needed to use this remote, I would have to log out and log in as the service account. It just seemed a bit messy.

https://seankilleen.com/2021/01/how-to-force-git-to-prompt-you-for-a-password/

**Comment by texsean** [comment]
nope, still errors... 

i was able to clone with password, but as soon as i reset the helper, its back to the password error.. mine also was manager.core bw.


C:\NETProjects>git config --system --unset credential.helper

C:\NETProjects>git fetch
Username for 'https://github.com': xxxx
Password for 'https://xxxxx@github.com':  ENTERED TOKEN

C:\NETProjects>git fetch
Username for 'https://github.com':
C:\NETProjects>git clone https://xxxxxxxxxxxxxxxxxxxxxxxxxxx.git /newnetprojects
Cloning into '/newnetprojects'...
Username for 'https://github.com': xxxxxxxxxxx
Password for 'https://xxxxx@github.com':
remote: Enumerating objects: 136700, done.
remote: Counting objects: 100% (7890/7890), done.
remote: Compressing objects: 100% (2645/2645), done.
remote: Total 136700 (delta 5305), reused 7542 (delta 5011), pack-reused 128810
Receiving objects: 100% (136700/136700), 508.96 MiB | 16.85 MiB/s, done.
Resolving deltas: 100% (99572/99572), done.
Updating files: 100% (22491/22491), done.

C:\NETProjects>git config --system credential.helper manager-core

C:\NETProjects>git fetch
remote: Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.
remote: Please see https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ for more information.
fatal: unable to access 'https://github.com/TheSiteControllerLLC/UTP_Development.git/': The requested URL returned error: 403

C:\NETProjects>cd \newnetprojects

C:\newnetprojects>git fetch
remote: Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.
remote: Please see https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ for more information.
fatal: unable to access 'https://github.com/TheSiteControllerLLC/UTP_Development.git/': The requested URL returned error: 403

C:\newnetprojects>git config --system --unset credential.helper

C:\newnetprojects>git fetch
Username for 'https://github.com': texsean
Password for 'https://texsean@github.com':

C:\newnetprojects>git config --system credential.helper manager-core

C:\newnetprojects>git fetch
remote: Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.
remote: Please see https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ for more information.
fatal: unable to access 'https://github.com/TheSiteControllerLLC/UTP_Development.git/': The requested URL returned error: 403

C:\newnetprojects>

**Comment by Keroxen** [comment]
Thank you, I was looking for something like this because I had to push to Umbraco Cloud and it was just saying auth failed.
